md/release-notes.md

```diff
@@ -205,6 +209,7 @@ has been removed.
 * BS-16996 External System Document not usable in file viewer widget or Overview
 * BS-17278 Cannot update custom widget property
 * BS-17526 Label property of input widget has not correct type
+* BS-16235 Action Remove from collection doesn't delete the correct item
```

Comment:
The link between the bug fix and the value sold in the article is not easy to make: the item removed wasn't the right one so now we offer you to keep editing the model even if the content is invalid?
I'm not sure I'm the right person to review this PR, because lots of it seems unclear to me.
Still, if my comments make sense to you, go ahead and clarify :)

Reply:
Do we allow ourselves to update the list of Bugs fixed in an already-released version?